app/src/main/java/org/jellyfin/androidtv/details/MovieDetailsFragment.kt

```diff
@@ -66,6 +68,14 @@ class MovieDetailsFragment(item: Movie) : BaseDetailsFragment<Movie>(item) {
 			item.cast.forEach(it::add)
 		}
 
+		val similarItems = TvApp.getApplication().apiClient.getSimilarItems(item)
+		val similarMovies = similarItems?.filter { it is Movie }
```

Comment:
Maybe add an expected type to the `getSimilarItems` so it won't respond with data you don't need

Reply:
Hmm, we could use generics and then integrate the filter into the method, but I'm not sure there how we would allow multiple types to pass.

Reply:
Doesn't the query class support setting a type?

Reply:
I don't think so

Reply:
Nope it does not support that.

Reply:
In that case just one-line it like this:
```kotlin
val similarItems = TvApp.getApplication().apiClient.getSimilarItems(item)?.filter { it is Movie }
```